README.md

```diff
@@ -90,7 +90,7 @@ to run the workflow on a high-performance computing (HPC) cluster.
 
 After installing Singularity, install the remaining dependencies with:
 ```bash
-mamba env create -f install/environment.yml
+conda env create -f install/environment.yml
```

Comment:
Just to be on the safe side, since most users still rely on conda.

Reply:
But does it make sense? we _do_ specify that mamba is a requirement<?>

Reply:
Lets leave the mamba

Reply:
Changed to mamba